### ISSUE TITLE
Advice on adding utterance boundary label (for dialogue) and inline tag for labeling heading (section in a passage and heading in a spreadsheet)

### ISSUE THREAD

**Issue by wailoktam** [google-research/bert#1120]
Hi, I am looking for some advice on using bert doing some summarization of dialogues into a spreadsheet or a passage with section headings. So my question is about 

A) adding boundary labels like  [CLS]  and;
B)  inline tags for headings like those used for named entity recognition or relation extraction.

For A)
I think adding boundaries between utterances of different parties in a dialogue would be helpful.  What is your suggestion on adding such a boundary label? 

1) Does it make sense to add it only during finetuning?
2) Would it have any positive impact if I add a couple of speech  corpora which can be added with such boundaries and use them together with the default corpus without such boundaries during pretraining?
3) I think using only speech corpora for pretraining would have negative impact on the language modelling. Do you agree?

For B)
Does it make sense to add it only during finetuning? I probably wont have a easy way to identify the headings in the corpus used during pretraining.  

In practice, these inline tags are added like custom vocabulary. Can you point me to resources for it?

Thank you.
